data-processed/UC3M-EpiGraph/metadata-UC3M-EpiGraph.txt

```diff
@@ -0,0 +1,9 @@
+team_name: Universidad Carlos III de Madrid
+model_name: EpiGraph simulator
+model_abbr: UC3M-EpiGraph
+model_contributors: David E. Singh (Universidad Carlos III de Madrid) <dexposit@inf.uc3m.es>, 
+Miguel Guzman (Universidad Carlos III de Madrid) <mguzman@pa.uc3m.es>, Maria Cristina Marinescu (Barcelona Supercomputing Center) <mariacristina.marinescu@gmail.com>
+website_url: https://www.arcos.inf.uc3m.es/epigraph/
```

Comment:
It looks like this line is causing the failing check for metadata. Please could you try removing `https://` in front of the web address? The colon character confuses the `yml` format that we use to machine-read this.